tests/modules/mcuboot/external_flash/Kconfig.sysbuild

```diff
@@ -5,6 +5,20 @@
 #
 
 config PM_EXTERNAL_FLASH_MCUBOOT_SECONDARY
-	default y if BOARD_NRF5340DK_NRF5340_CPUAPP || BOARD_NRF52840DK_NRF52840
+	default y if SOC_NRF5340_CPUAPP
```

Comment:
this is removing support for nrf52840dk?

Reply:
right, to much copy-paste. Thank you!